packages/dashboard/src/pages/Wallets/components/Assets/index.tsx

```diff
@@ -85,15 +80,16 @@ export const Assets = memo<TokenAssetsProps>(({ network }) => {
                                 size="small"
                                 color="secondary"
                                 className={classes.addCustomTokenButton}
-                                onClick={() =>
-                                    currentTab === AssetTab.Token
-                                        ? setSelectToken({
-                                              open: true,
-                                              uuid: id,
-                                              FungibleTokenListProps: { whitelist: [] },
-                                          })
-                                        : setAddCollectibleOpen(true)
-                                }>
+                                onClick={async () => {
+                                    if (currentTab === AssetTab.Token) {
+                                        // TODO handle result
+                                        await pickToken({
+                                            whitelist: [],
+                                        })
+                                    } else {
+                                        setAddCollectibleOpen(true)
+                                    }
+                                }}>
```

Comment:
```suggestion
                                onClick={async () => {
                                    switch(currentTab) {
                                          // ...
                                    }
                                }}>
```

Reply:
It's just a specified handle for `AssetTab.Token`.